command/agent/fs_endpoint.go

```diff
@@ -194,11 +194,14 @@ func (s *HTTPServer) FileCatRequest(resp http.ResponseWriter, req *http.Request)
 // Stream streams the content of a file blocking on EOF.
 // The parameters are:
 // * path: path to file to stream.
+// * follow: A boolean of whether to follow the file, defaults to true.
 // * offset: The offset to start streaming data at, defaults to zero.
 // * origin: Either "start" or "end" and defines from where the offset is
 //           applied. Defaults to "start".
 func (s *HTTPServer) Stream(resp http.ResponseWriter, req *http.Request) (interface{}, error) {
 	var allocID, path string
+	var follow bool = true
```

Comment:
This one can be and keep it closer to the block with `followStr`.  It's more conventional in go to keep variable initializations closer to where they are first used.  This function uses this pattern for `offset`, but doesn't for `allocID` and `path` 🤷‍♂

```suggestion
	follow := true
```